src/Stoa.ts

```diff
@@ -21,11 +22,64 @@ class Stoa {
             res.send(req.params.address + ':' + req.query.height);
         });
 
+        /**
+         * When a request is received through the `/push` handler
+         * JSON block data is parsed and stored on each storage.
+         */
         this.stoa.post("/push",
             (req: express.Request, res: express.Response, next: express.NextFunction) => {
-            console.log(req.body);
-            res.status(200).send();
-        });
+            var data: any = req.body;
```

Comment:
Can you just use `req.body` directly since it's only used twice ? [Follow the rule of three](https://blog.codinghorror.com/rule-of-three/).

Reply:
Delete Unnecessary Variables